mp/src/game/server/neo/neo_player.cpp

```diff
@@ -489,7 +489,13 @@ void CNEO_Player::Precache( void )
 
 void CNEO_Player::Spawn(void)
 {
-	ShowViewPortPanel(PANEL_SPECGUI, (GetTeamNumber() == TEAM_SPECTATOR ? true : false));
+	int teamNumber = GetTeamNumber();
+	ShowViewPortPanel(PANEL_SPECGUI, (teamNumber == TEAM_SPECTATOR ? true : false));
```

Comment:
`ShowViewPortPanel(PANEL_SPECGUI, teamNumber == TEAM_SPECTATOR);` ?

Reply:
im gonna do this one in the web editor, wish me luck